docs/groovy-script/mods/abyssaltweaker/fuel.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Fuel"
+titleTemplate: "Abyssal Tweaker | CleanroomMC"
+description: "Allows configuring the items that can be used as Transmutator and/or Crystallizer fuels."
+source_code_link: "https://github.com/CleanroomMC/GroovyScript/blob/master/src/main/java/com/teamdimensional/abyssaltweaker/compat/groovyscript/Fuel.java"
+---
+
+# Fuel (Abyssal Tweaker)
+
+## Description
+
+Allows configuring the items that can be used as Transmutator and/or Crystallizer fuels.
```

Comment:
i think splitting this into `TransmuterFuel` and `CrystallizerFuel` would be a better way to organize this. no big deal either way.

Reply:
🥺